include/zephyr/bluetooth/audio/audio.h

```diff
@@ -1271,45 +1278,61 @@ void bt_audio_stream_cb_register(struct bt_audio_stream *stream,
  * @{
  */
 
-/** @brief Register unicast server callbacks.
+/** @brief Register Publish Audio Capabilities Service callbacks.
  *
  *  Only one callback structure can be registered, and attempting to
  *  registering more than one will result in an error.
  *
- *  @param cb  Unicast server callback structure.
+ *  This can only be done for the Unicast Server
+ *  (@kconfig{CONFIG_BT_AUDIO_UNICAST_SERVER}) and Broadcast Sink
+ *  (@kconfig{CONFIG_BT_AUDIO_BROADCAST_SINK}) roles.
  *
- *  @return 0 in case of success or negative value in case of error.
- */
-int bt_audio_unicast_server_register_cb(const struct bt_audio_unicast_server_cb *cb);
-
-/** @brief Unregister unicast server callbacks.
- *
- *  May only unregister a callback structure that has previously been
- *  registered by bt_audio_unicast_server_register_cb().
+ *  Calling bt_audio_capability_register() will implicitly register the
+ *  callbacks.
  *
  *  @param cb  Unicast server callback structure.
  *
  *  @return 0 in case of success or negative value in case of error.
  */
-int bt_audio_unicast_server_unregister_cb(const struct bt_audio_unicast_server_cb *cb);
+int bt_audio_pacs_register_cb(const struct bt_audio_pacs_cb *cb);
 
-/** @brief Notify location changed
- *
- * Notify connected clients that the location has changed
+/** @brief Notify that the location has changed
  *
- * @param dir       Direction of the endpoint.
+ * @param dir Direction of the location changed.
  *
  * @return 0 in case of success or negative value in case of error.
  */
-int bt_audio_unicast_server_location_changed(enum bt_audio_dir dir);
+int bt_audio_pacs_location_changed(enum bt_audio_dir dir);
```

Comment:
Maybe outside of this PR, so possibly make a note of it.
Does this call notify a change of the locations, or just of the direction of the location?  In the latter case, the name is probably confusing.